dsfs/dataset.go

```diff
@@ -527,13 +527,29 @@ func WriteDataset(store cafs.Filestore, ds *dataset.Dataset, dataFile cafs.File,
 		adder.AddFile(stf)
 	}
 
-	if ds.VisConfig != nil {
-		vc, err := JSONFile(PackageFileVisConfig.String(), ds.VisConfig)
-		if err != nil {
-			return datastore.NewKey(""), fmt.Errorf("error marshaling dataset visconfig to json: %s", err.Error())
+	if ds.Viz != nil {
+		// vc, err := JSONFile(PackageFileViz.String(), ds.Viz)
+		// if err != nil {
+		// 	return datastore.NewKey(""), fmt.Errorf("error marshaling dataset viz to json: %s", err.Error())
+		// }
+		// fileTasks++
+		// adder.AddFile(vc)
+		if ds.Viz.Script != nil {
+			fileTasks++
+			adder.AddFile(cafs.NewMemfileReader("viz_script", ds.Viz.Script))
```

Comment:
Change into a constant vizScriptTask, same as the case statement down below that looks for it. Would be nice for transform_script as well, but that can be done later.